docs/timelines.md

```diff
@@ -37,7 +37,7 @@ The following types of trigger can be created. You can record these actions in r
 or enter them as manual triggers at specified times using the touch screen.
 
 Trigger             | Action
---------------------|----------
+---|-----
```

Comment:
I was a bit confused by this. Turns out the pandoc pipe tables format assigns meaning to the relative number of dashes in the separator row when cells have lots of content (who knew - see `pipe_tables` on https://pandoc.org/MANUAL.html#pandocs-markdown ). Perhaps we should set `--columns` or adjust the input formatting as appropriate?

Might also be worth leaving a comment in the source so this doesn't trip up someone modifying this in the future.

Reply:
It seems tables are a bit of a nightmare in markdown, everything interprets them differently. This could affect other tables in the manual as we have tried to format the tables to be easily readable in the markdown so the number of dashes may vary..

Reply:
I believe @garethnunns went through and fixed the other tables already. The only ones that I noticed that were broken in the pdf were the ones in the timeline section.